certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -178,6 +178,52 @@
           }
         ]
       }
+    },
+	"/GetDocxBookmarks": {
```

Comment:
@stsiek this one still has problems with indentation, there are extra spaces. Please take a look one more time.